src/braket/device_schema/dwave/dwave_advantage_device_parameters_v1.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import Field
+
+from braket.device_schema.dwave.dwave_advantage_device_level_parameters_v1 import (
+    DwaveAdvantageDeviceLevelParameters,
+)
+from braket.schema_common import BraketSchemaBase, BraketSchemaHeader
+
+
+class DwaveAdvantageDeviceParameters(BraketSchemaBase):
```

Comment:
We are adding all these classes but have not included tests for any of them. Could we please add corresponding tests for all the new files in 
amazon-braket-schemas-python/test/unit_tests/braket/device_schema/dwave ?